pkg/occlient/occlient.go

```diff
@@ -57,6 +57,32 @@ import (
 	"k8s.io/client-go/util/retry"
 )
 
+// CreateType is an enum to indicate the type of source of component -- local source/binary or git for the generation of app/component names
+type CreateType string
```

Comment:
This is used only in `component` if I'm looking correctly. Why was this moved here?